packages/codec/lib/abi-data/allocate/index.ts

```diff
@@ -650,22 +721,33 @@ function getCalldataAllocationsForContract(
 }
 
 //note: returns undefined if undefined is passed in
+//for first argument
 function defaultConstructorAllocation(
-  constructorContext: Contexts.DecoderContext
-): CalldataAllocation | undefined {
+  constructorContext: Contexts.DecoderContext,
+  deployedContext: Contexts.DecoderContext
+): CalldataAndReturndataAllocation | undefined {
   if (!constructorContext) {
     return undefined;
   }
   let rawLength = constructorContext.binary.length;
   let offset = (rawLength - 2) / 2; //number of bytes in 0x-prefixed bytestring
-  return {
+  let input = {
     offset,
     abi: AbiDataUtils.DEFAULT_CONSTRUCTOR_ABI,
     arguments: [] as CalldataArgumentAllocation[],
-    allocationMode: "full"
+    allocationMode: "full" as const
   };
+  let output = CONSTRUCTOR_OUTPUT_ALLOCATION;
```

Comment:
```suggestion
  conost output = constructorOutputAllocation;
```

Reply:
Well I can't commit this suggestion, you misspelled a keyword :P

(...but I'm just going to mark this resolved for simplicity, we can discuss it on the other one.)

Reply:
Ugh my MacBook butterfly keys!!